### DISCUSSION TITLE
Just a heads-up - Problems ahead with RestSharp?

### DISCUSSION THREAD

**Discussion by wheelersf** [abuzuhri/Amazon-SP-API-CSharp#69] [General]
Just saw this on one of the feeds I subscribe to.  Points out a breaking change with version 13.0.1:

[https://www.codeproject.com/Lounge.aspx?fid=1159&df=90&mpp=100&sort=Position&spc=Relaxed&tid=5854170](url)

**Comment by abuzuhri** [comment]
The impact for us will be only internal our library and will not take too much time to do
